inst/resources/environment.py

```diff
@@ -141,28 +194,32 @@ def pip_freeze():
     """
     try:
         proc = subprocess.Popen(
-            [sys.executable, '-m', 'pip', 'freeze'],
-            stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+            [sys.executable, "-m", "pip", "list", "--format=freeze"],
```

Comment:
Do we know when the `--format=freeze` argument was added? Do we need to worry about older versions of `pip` that don't support this flag?

Reply:
It is present in pip 9.0 which was released in November 2016. We're probably OK?